sources.list

```diff
@@ -1,27 +1,27 @@
 # See http://help.ubuntu.com/community/UpgradeNotes for how to upgrade to
 # newer versions of the distribution.
 
-deb http://archive.ubuntu.com/ubuntu/ trusty main restricted
-# deb-src http://archive.ubuntu.com/ubuntu/ trusty restricted main multiverse universe
+deb http://archive.ubuntu.com/ubuntu/ xenial main restricted
```

Comment:
`yakkety` has the systemd compatibility as well besides it is more faster and more advanced than `xenial` and the there's an issue in the repo which states to update the script to `yakkety` 

The main "switch" is being done in the Exec= lines of the
corresponding /usr/share/xsessions/*.desktop. Display managers use
these to show which sessions are available and to know how to start
them. This happend in ubuntu-desktop 3.18.1.2-1ubuntu5 and
xubuntu-default-settings 16.10., which just landed in Yakkety.

The upstart to systemd transition depends on/contains a separate
transition from a session-centric to user-centric D-Bus daemon and
services. A lot of D-Bus services like pulseaudio, gvfs, SSH agents,
keyring daemon etc. are useful to other parallel (non-graphical, i. e.
VT or ssh) logins of the same user, so these will now continue to live
until the last session of a particular user gets stopped. This is being achieved by installing the
"dbus-user-session" package as you know, on which the converted
ubuntu-session/xubuntu-default-settings packages now depend.  This
also caused some fallout mostly in policykit, gnome-keyring, and
AppArmor profiles, which all got fixed in Yakkety